packages/jest-leak-detector/src/index.ts

```diff
@@ -26,23 +28,33 @@ export default class LeakDetector {
       );
     }
 
-    let weak: typeof import('weak-napi');
+    if (globalThis.FinalizationRegistry) {
```

Comment:
I see it's supported since node 14. Could you add a `TODO` comment or something to remove the `if` when we drop node 12?

Reply:
> I see it's supported since node 14. Could you add a `TODO` comment or something to remove the `if` when we drop node 12?

Oh, that's true! The TODO was added.